src/app/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from "@angular/core"
+
+@Injectable({
+    providedIn: 'root'
+  })
+
+export class Constants  {
+    tasksApiUrl = "http://localhost:3000/tasks/"
```

Comment:
You don't need to create a class for the constants and inject it as a dependency, just create this file and export each constant, for example:
export const tasksApiUrl = etcetc;